ibm/Dockerfile.j2

```diff
@@ -0,0 +1,33 @@
+FROM python:{{ BASE_IMAGE_VERSION }}-slim-buster
+
+LABEL maintainer="Maxime Sibellas <maxime.sibellas@ekino.com>"
+
+ARG TERRAFORM_VERSION
+ARG IBM_PROVIDER_VERSION
+ARG IBM_CLI_VERSION
+
+# ENV PATH=$PATH:/usr/local/bin
+
+WORKDIR /builds
+
+RUN pip install -U pip \
+    && pip install awscli boto3 pipenv
+
+RUN apt-get update -qq \
+    && apt-get -y -qq install wget \
+    && echo "Install IBMCloud CLI" && \
+    wget https://public.dhe.ibm.com/cloud/bluemix/cli/bluemix-cli/${IBM_CLI_VERSION}/binaries/IBM_Cloud_CLI_${IBM_CLI_VERSION}_linux_amd64.tgz && \
+    tar -xzf IBM_Cloud_CLI_${IBM_CLI_VERSION}_linux_amd64.tgz && \
+    mv IBM_Cloud_CLI/ibmcloud /usr/local/bin && \
+    rm -rf IBM_Cloud_CLI_${IBM_CLI_VERSION}_linux_amd64.tgz IBM_Cloud_CLI \
+    && apt-get -y -qq remove wget && apt-get -y -qq autoremove
+RUN apt-get -y -qq install unzip wget \
```

Comment:
`remove wget` then `install wget`? 🤔 

Can all the utilities be added at the beginning of the script, then all removed at the end?

Do we really gain something by using 2 layers? The files downloaded by `apt-get update -qq` are not removed, so the first layer is larger than it could be.

Reply:
It was a try to clean the image. I made multiple layers for developping the image and fasten its build in local environment.
After checking, its almost lightest to not clean apt cache but I kept it.